proto/public/api.proto

```diff
@@ -159,6 +159,8 @@ message BasicStats {
   uint64 latency_ms_p50 = 3;
   uint64 latency_ms_p95 = 4;
   uint64 latency_ms_p99 = 5;
+  // count of requests that are TLSed
+  uint64 secured_request_count = 6;
```

Comment:
I think, from the API point of view, these should be considered "meshed" and not secure. I really dislike the word "secure", as it's not at all clear what it means

Reply:
Specifically, there are a wide variety of aspects of security. Encrypting the connection is different from validating the peer, for instance. To avoid getting into this level of detail at this point of the API, i think we should talk only about what we know we can be authoritative about.